tensorflow/contrib/rnn/python/ops/rnn_cell.py

```diff
@@ -2097,7 +2097,7 @@ def __init__(self,
 
     state_size = tensor_shape.TensorShape(
         self._input_shape[:-1] + [self._output_channels])
-    self._state_size = rnn_cell_impl.LSTMStateTuple(state_size, state_size)
+    self._state_size = rnn_cell_impl.LSTMStateTuple(state_size, self._output_size)
     self._output_size = tensor_shape.TensorShape(
```

Comment:
Isn't output size defined after this line?

Reply:
Oh sorry. I will correct that now

Reply:
Please review the updated patch